src/components/Header.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */ 
+import React from 'react';
```

Comment:
- [ ] Kindly enable the linter in this file. Linter enforces quality code standards and therefore should not be disabled.